src/trace-api.ts

```diff
@@ -75,36 +76,37 @@ export class TraceAgent implements TraceAgentInterface {
   public readonly constants = Constants;
   public readonly labels = TraceLabels;
 
-  private pluginName_: string;
-  private logger_: Logger;
-  private config_: TraceAgentConfig;
+  private pluginName: string;
```

Comment:
This is not semver major?

Reply:
`pluginName_` is an undocumented field. I would not consider this semver-major.